modules/security_groups/main.tf

```diff
@@ -20,16 +20,30 @@ resource "aws_security_group" "vpc-postgres-ingress-all-egress" {
     to_port   = "5432"
     protocol  = "TCP"
     self      = true
+    description = "Allow PostgreSQL ingress for ${var.prefix} Orca lambdas."
   }
 
-  ## This is added to allow all egress for lambdas. See Terraform documentation
+  ## This is added to allow all egress for Orca lambdas. See Terraform documentation
   ## for egress everything notes.
   egress {
     from_port        = 0
     to_port          = 0
     protocol         = "-1"
     cidr_blocks      = ["0.0.0.0/0"]
     ipv6_cidr_blocks = ["::/0"]
+    description = "Allow all egress for ${var.prefix} Orca lambdas."
   }
 
 }
+
+## rds_allow_lambda_access - PostgreSQL Security Group Rule
+## ==============================================================================
+resource "aws_security_group_rule" "rds_allow_lambda_access" {
+  type                     = "ingress"
+  from_port                = 0
```

Comment:
I wonder if we shouldn't put a comment in that we have "0" here since we are designating the protocol of "all". I wonder if this could be tightened up? Maybe we can narrow to "tcp" protocol and narrow the port range? It would take some testing to make sure everything would still work okay.

Reply:
Reset to our previous postgres rule. Ran a couple basic tests, still seems functional.